### PR TITLE
V3.0.0

### PR DESCRIPTION
**Pull request by kagg-design**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@master
```
Comment (e271828-) [inline .github/workflows/ci.yml:32]: do we really want to track master here?

File: readme.txt [hunk 2]
```diff
@@ -471,6 +482,16 @@ Instructions for popular native integrations are below:
 
 == Changelog ==
 
+= 3.0.0 =
+* Dropped support of PHP 5.6. Minimum required PHP version is 7.0 now.
```
Comment (e271828-) [inline readme.txt:486]:
```suggestion
* Dropped support for PHP 5.6. Minimum required PHP version is now 7.0.
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kagg-design) [reply .github/workflows/ci.yml]: We needed to go higher from v2, as they used an outdated node version of Node, and GitHub Actions reported it. You are right; using a stable release is safer, so I put v3 and see no warnings about the Node version now. Will improve `set-output` related stuff later.

File: .github/workflows/create-zip.yml [hunk 3]
```diff
@@ -11,14 +11,14 @@ jobs:
...
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Get Composer cache directory
         id: composer-cache
         run: echo "::set-output name=dir::$(composer config cache-files-dir)"
 
       - name: Set up Composer caching
-        uses: actions/cache@v2
+        uses: actions/cache@master
```
Comment (e271828-) [inline .github/workflows/create-zip.yml:21]:
```suggestion
        uses: actions/cache@v3
```

**Review by e271828- — APPROVED** [review]

**Pull request merged**